build/cwl-wrappers/cmo-marianas/0.0.0/cmo-marianas.original.cwl

```diff
@@ -0,0 +1,29 @@
+cwlVersion: cwl:v1.0
+
+class: CommandLineTool
+
+baseCommand: [cmo_marianas]
+
+arguments: ["-server", "-Xms8g", "-Xmx8g", "-cp"]
+
+doc: |
+  None
+
+inputs:
+  r1_fastq:
+    type: File
+    inputBinding:
+      prefix: --r1_fastq
+    secondaryFiles:
+    - ${self.location.replace("_R1_", "_R2")}
+    - ${self.location.replace(/(.*)(\/.*$)/, "$1/SampleSheet.csv")}
```

Comment:
Wasn't sure if this was the best way to reference files in the same directory, but with different names.